### ISSUE TITLE
ZPermissions merging

### ISSUE THREAD

**Issue by hcw70** [UnfixableExploit/zPermissions#1]
I am using (original) zPermissions for my multi server bungeecord / spigot 1.10.2 setup, but found out recently that the original zPermission seems to be unmaintained and that there are some forks (as yours) on Github.

I am having however some problems with my setup (i.e. bungeeChat and PermissionSync dont work).

Do you intend to be the "new maintainer" of zPermissions? Are there any plans to merge the forks on github?

Kind regards

Hauke

**Comment by Developer0fTruth** [comment]
Yeah this is my other account, yeah at the time I was going to have a version maintained for myself and others should they want to use it, currently the PC I was using for development is now long gone. I'm planing to restart development for this soon, just got to wait saving up money for a new system :). I don't intend to be the new maintainer, just plan on making sure its works on the version of Minecraft I plan to use on my own private server but i'll port it to the other versions of 1.10 and up for people who request it
